.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - main
+  push:
+    paths:
+      - 'dev/**.md'
```

Comment:
We are pushing in `src/**.md`. Also, we could have some pushes in `src/` that are not necessarily `.md` and should be previewed as well.